lib/garbage_collector.rb

```diff
@@ -60,11 +68,17 @@ def run(projects)
         end
       end
     end
+
+    unless projects_not_found.empty?
+      log("\nGarbage collection was unsuccessful. Could not find projects:", forced: true)
```

Comment:
I went for adding a `forced` flag to `log` instead of using `puts` directly to keep all the logging centralized in one method.